tools/_shared/c_header_parser/tilemaps.h

```diff
@@ -1,308 +0,0 @@
-#ifndef GUARD_TILEMAPS
```

Comment:
Why was this removed?

Reply:
Look at the path it's in!
Wasn't supposed to be there in the first place! 😋